package.json

```diff
@@ -41,7 +42,11 @@
     "format": "prettier --write .",
     "format:check": "prettier --check .",
     "tsc": "tsc",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "check:audit": "check-audit",
+    "audit:resolve": "resolve-audit",
+    "check:lint": "eslint src --ext .ts",
```

Comment:
Can we add in a `check` command like we have on ads-serve and the admin dashboard?  In a later PR is fine.